postgresql/provider.go

```diff
@@ -236,6 +270,13 @@ func validateExpectedVersion(v interface{}, key string) (warnings []string, erro
 	return
 }
 
+func validateParsableDuration(v interface{}, key string) (warnings []string, errors []error) {
```

Comment:
nit: `any`, it's not 2022:
```suggestion
func validateParsableDuration(v any, key string) (warnings []string, errors []error) {
```